### PR TITLE
Accounts section

### LINKED ISSUE #4: Console Providers section

### PR DESCRIPTION
**Pull request by r2dedios**
Closes #4

### CODE REVIEW

File: src/app/Overview/Overview.tsx [hunk 1]
```diff
@@ -159,6 +159,47 @@ const AggregateStatusCards: React.FunctionComponent = () => {
           layout: "multiIcon",
         }
       ],
+      accountsPerProvider: [
+        {
+          title: "AWS Accounts",
+          content: [
+            {
+              count: clusterCounts["AWS"] || 0,
```
Comment (vbelouso) [inline src/app/Overview/Overview.tsx:167]: This code calculates the number of clusters per account.
In the `clusterPerCP.accounts.forEach((account)` loop we can have something like
```typescript
 accountsPerCP[provider] = (accountsPerCP[provider] || 0) + 1;
 ```
 and then use it
 ```typescript
 accountsPerProvider: [
      {
        title: 'AWS Accounts',
        content: [
          {
            count: accountsPerCP['AWS'] || 0,
            icon: <AwsIcon color="var(--pf-v5-global--danger-color--100)" />,
            ref: '/accounts?provider=AWS',
          },
        ],
        layout: 'multiIcon',
      }
 ```

File: src/app/AccountDetails/AccountDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            height: "100vh",
+          }}
+        >
+          <Spinner size="xl" />
+        </div>
+      ) : (
+        <Table aria-label="Simple table">
```
Comment (vbelouso) [inline src/app/AccountDetails/AccountDetails.tsx:85]: When I click on the sidebar menu`Accounts` - `<account_name>` - `Clusters` tab the layout is broken
Screenshot from 2023-11-28 09-52-34 - 29771932

File: src/app/Overview/Overview.tsx (hunk 1 above)
Comment (r2dedios) [reply src/app/Overview/Overview.tsx:167]: Fixed id this PR #15

File: src/app/AccountDetails/AccountDetails.tsx (hunk 2 above)
Comment (r2dedios) [reply src/app/AccountDetails/AccountDetails.tsx:85]: Fixed in this PR #15 
image

**Pull request merged**